### DISCUSSION TITLE
What is the difference between "expression" and "value"?

### DISCUSSION THREAD

**Discussion by Worldwidebrine** [mdn/mdn-community#567] [Content]
What is the difference between "expression" and "value"?
They are similar.
In my opinion, "expression" is the name before evaluation and "value" is the name after evaluation. Am I wrong?

**Comment by Josh-Cena** [comment]
You are right. `1 + 1` is an expression (namely, an additive expression consisting of two literal expressions), and its value is the number 2. `0xa` is a literal expression whose value is the number 10.

(I actually planned to incorporate this explanation into the glossary entries a few days ago, just haven't started on it yet)
